backend/vendor/assets/stylesheets/jquery.jstree/themes/spree/style.scss

```diff
@@ -0,0 +1,216 @@
+/*
+ * jsTree Spree theme 1.0
+ */
+
+.jstree-spree {
+
+  li, li ul {
+    border-color: #ededed;
+    border-style: solid;
+  }
+
+  li {
+    margin-left: 26px;
+    border-width: 0 0 1px 0;
+
+    &:last-of-type {
+      border-bottom-width: 0;
+    }
+
+    ul {
+      border-width: 1px 0 0 0;
+    }
+  }
+
+  ins {
+    position: relative;
+    cursor: pointer;
+    width: 24px;
+    line-height: 1;
+    height: 38px;
+    vertical-align: middle;
+    font-family: 'Glyphicons Halflings';
+    color: #555;
+
+    &:before {
+      position: absolute;
+      top: 12px;
+      padding-left: 4px;
+      font-size: 12px;
+    }
+  }
+
+  .jstree-open > ins:before { content: '\e114' }
+
+  .jstree-closed > ins:before { content: '\e080' }
+
+  .jstree-leaf > ins {
+    background-position: -36px 0;
+  }
+
+  .jstree-locked a {
+    color: silver;
+    cursor: default;
+  }
+
+  a {
+    line-height: 38px;
+    height: 38px;
+    padding: 0;
+    color: #555;
+    cursor: context-menu;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    user-select: none;
+
+    &.jstree-search {
+      color: aqua;
+    }
+
+    &.jstree-loading .jstree-icon {
+      background: image-url("jquery.jstree/themes/spree/throbber.gif") center center no-repeat !important;
+
+      &:before { content: '' }
+    }
+
+    .jstree-icon {
+      vertical-align: middle;
+      font-family: 'Glyphicons Halflings';
+      line-height: 20px;
+      width: 20px;
+      height: 24px;
+      text-align: center;
+      margin-right: 8px;
+      pointer-events: none;
+
+      &:before {
+        content: "\e118";
+        top: 1px;
+        padding-left: 0;
+      }
+    }
+  }
+
+  .jstree-rename-input {
+    height: 38px !important;
+    line-height: 38px !important;
+    left: 46px !important;
+    padding: 0 5px !important;
+  }
+
+  .jstree-hovered {
+    background: #e7f4f9;
+    border: 1px solid #d8f0fa;
+    padding: 0 3px 0 1px;
+    text-shadow: 1px 1px 1px silver;
```

Comment:
Color `silver` should be written in hexadecimal form as `#c0c0c0`<br>Color literals like `silver` should only be used in variable declarations; they should be referred to via variable everywhere else.